attestation-manager/src/cmd/app/main.go

```diff
@@ -218,7 +218,29 @@ func retryOperation(operation func() (bool, interface{}), retryCount int, retryI
 	return false, nil
 }
 
-func cordonDrainNode() {
+func handleErrorDetails(cfg *constants.Config, attestToken string) {
```

Comment:
this function name doesn't really match the function logic, it might cause confusion for a reader that is just skimming the code.  I suggest renaming it to some more descriptively accurate like logFlavorTrustReportOnError()...

Reply:
we will rename the as per fucntionality.